kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ConsumerGroupsResourceIntegrationTest.java

```diff
@@ -107,12 +110,58 @@ public void listConsumerGroups_returnsConsumerGroups() {
                             .build()))
                 .build());
 
+    ListConsumerGroupsResponse expected2 =
+        ListConsumerGroupsResponse.create(
+            ConsumerGroupDataList.builder()
+                .setMetadata(
+                    ResourceCollection.Metadata.builder()
+                        .setSelf(baseUrl + "/v3/clusters/" + clusterId + "/consumer-groups")
+                        .build())
+                .setData(
+                    singletonList(
+                        ConsumerGroupData.builder()
+                            .setMetadata(
+                                Resource.Metadata.builder()
+                                    .setSelf(
+                                        baseUrl
+                                            + "/v3/clusters/"
+                                            + clusterId
+                                            + "/consumer-groups/consumer-group-1")
+                                    .setResourceName(
+                                        "crn:///kafka="
+                                            + clusterId
+                                            + "/consumer-group=consumer-group-1")
+                                    .build())
+                            .setClusterId(clusterId)
+                            .setConsumerGroupId("consumer-group-1")
+                            .setSimple(false)
+                            .setPartitionAssignor("range")
+                            .setState(State.STABLE)
```

Comment:
nit: This seems to be the only difference from `expected1` - let's use a helper method for the creation of the responses, this will make the code shorter and it will make the difference obvious.